src/resources/boards/board.service.js

```diff
@@ -0,0 +1,64 @@
+const { v4: uuidv4 } = require('uuid');
+const path = require("path");
+const { writeDataToFile } = require('../../utils/fileService');
+const { badRequest, notFound } = require('../../messages/errors');
+
+const dataWay = path.resolve("./data/boards.json")
+// eslint-disable-next-line import/no-dynamic-require
+const boardData = require(dataWay)
+
+
+
+class BoardService {
+    static async getAll() {
+        const boards = await boardData;
+        return boards;
+    }
+
+    static async create(res, board) {
+        try{
+            const newBoard = await { id: uuidv4(), ...board };
+            boardData.push(newBoard)
+            writeDataToFile(dataWay, boardData)
```

Comment:
better make writeDataToFile return promise that will be resolved when file is written, and use async function to write file. 
so you'll need await if it will be async